File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-1.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int n = nums.length;

        if(n == 1){
            return 1;
        }

        int left = 1;
        int right = 1;
        
        while(right < n){
            while(right < n && nums[right-1] == nums[right]){
                right++;
            }

            if(right < n){
            nums[left] = nums[right];
            }
            else{
                return left;
            }

            left++;
            right++;
        }

        return left;

    }
}

File: Data Structures & Algorithms/palindromic-substrings/submission-1.py
class Solution:
    def countSubstrings(self, s: str) -> int:
        # Odd count
        count = 0
        for i in range(len(s)):
            left, right = i, i
            while left >= 0 and right < len(s):
                if s[left] != s[right]:
                    break
                count += 1
                left -= 1
                right += 1
                
        
        # Even count
        for i in range(len(s)-1):
            left, right = i, i+1
            while left >= 0 and right < len(s):
                if s[left] != s[right]:
                    break
                count += 1
                left -= 1
                right += 1
        
        return count